chainntnfs/txnotifier_test.go

```diff
@@ -331,27 +311,11 @@ func TestTxNotifierFutureConfDispatch(t *testing.T) {
 	// We should not receive any event notifications for tx1 since it has
 	// already been confirmed.
 	select {
-	case <-ntfn1.Event.Updates:
-		t.Fatal("Received unexpected confirmation update for tx1")
 	case txConf := <-ntfn1.Event.Confirmed:
 		t.Fatalf("Received unexpected confirmation for tx1: %v", txConf)
 	default:
 	}
 
-	// We should only receive one update since the last at the new height,
-	// indicating how many confirmations are still left.
-	select {
-	case numConfsLeft := <-ntfn2.Event.Updates:
-		const expected = 0
-		if numConfsLeft != expected {
-			t.Fatalf("Received incorrect confirmation update: tx2 "+
-				"expected %d confirmations left, got %d",
-				expected, numConfsLeft)
-		}
-	default:
-		t.Fatal("Expected confirmation update for tx2")
-	}
-
 	// A confirmation notification for tx2 should be dispatched, since it
```

Comment:
To improve the test it would be great if you could check that neither `ntfn1` nor `ntfn2` receives another confirmation event after this check. This could use a select which waits a short time after it unblocks.